Std/Tactic/TryThis.lean

```diff
@@ -75,65 +339,135 @@ partial def replaceMVarsByUnderscores [Monad m] [MonadQuotation m]
 def delabToRefinableSyntax (e : Expr) : TermElabM Term :=
   return ⟨← replaceMVarsByUnderscores (← delab e)⟩
 
+/-- Delaborate `e` into a suggestion suitable for use in `refine`. -/
+def delabToRefinableSuggestion (e : Expr) : TermElabM Suggestion :=
+  return { suggestion := ← delabToRefinableSyntax e, messageData? := e }
+
+/-! # Widget hooks -/
+
+/-- Core of `addSuggestion` and `addSuggestions`. Whether we use a `Try This:` display or a
+`Try These:` display is controlled by `isSingular`. -/
```

Comment:
Couldn't `isSingular` just be replaced by `suggestions.size = 1`?

Reply:
Not without changing the styling: that would change a list with one element to a non-list. `addSuggestions` with one suggestion would then produce
```
Try these: rfl
```
instead of

`Try these:`

* `rfl`

The first option looks weird—at least to me—but the second doesn't.

Reply:
(Renamed `isSingular` to `isInline` in 0023326, because however this resolves, that's a better name. :) )

Reply:
Okay, I understand now. That's good as is, and the rename is helpful!